src/main/java/redis/clients/jedis/BinaryJedisCluster.java

```diff
@@ -270,21 +270,21 @@ public Long execute(Jedis connection) {
   }
 
   @Override
-  public String psetex(final byte[] key, final long milliseconds, final byte[] value) {
```

Comment:
I don't understand the changes in this file. Seems like you're just renaming things but nothing is actually being added / removed?. What's the purpose of the changes here?

Reply:
I have placed `psetex` after `setex`. Yes, this change does not have any implementation value.

But in other interfaces and classes (except this one) `setex` places earlier than `psetex`. I thought it to be a good idea to maintain it here too.

I definitely can change this if you oppose this idea.

Reply:
I'm OK to arrange the sequence of methods, more clearly, IMHO ideally we should.

Reply:
It seems code re-arranging makes merging harder. So undoing those changes.